Makefile

```diff
@@ -1,5 +1,6 @@
 GO=$(shell which go)
 OTELCOL_VERSION ?= 0.95.0
+OCB_VERSION ?= 0.93.0
```

Comment:
Why are we splitting the OCB and OTELcol versions?

Reply:
The `ocb` tools defines the used go version. Since the current ubi comes with go 1.20, it will fail when we try to build the distro using 0.95.0.

Reply:
Could you please put a comment here to revert it back once the base image supports go 1.21? (ubi 8.10) ?

Reply:
Added a comment.